### PR TITLE
feat(@leav/ui): Values list with add

### PR DESCRIPTION
**Pull request by jrmyb**
## Checklist

Definition Of Review

-   [x] Own code review done (add notes for others)
-   [ ] Write message in teams channel
    ```
     <Title>

    *️⃣ Impacted projects : Core - DataStudio - Admin - @leav/ui - @leav/utils - ...

    📖 Ticket: https://aristid.atlassian.net/browse/<JIRA_TICKET_IDENTIFIER>

    🧑‍💻 PR: <link to PR/MR>

    ℹ Info: <brief explanation - context - how to test>
    ```

Definition Of Mergeable

-   [ ] 2 approves
-   [ ] 1 functional review - US has been tested
-   [ ] Every comment is handled - blocking ones have been resolved by reviewer
-   [ ] Design OK
-   [ ] Can be tested by POs
-   [ ] PR was introduced during daily meeting

### CODE REVIEW

File: apps/data-studio/src/_gqlTypes/index.ts [hunk 1]
```diff
@@ -804,13 +806,13 @@ export type ValueBatchInput = {
...
     metadata?: InputMaybe<Array<InputMaybe<ValueMetadataInput>>>;
-    value?: InputMaybe<Scalars['String']>;
+    payload?: InputMaybe<Scalars['String']>;
 };
 
 export type ValueInput = {
     id_value?: InputMaybe<Scalars['ID']>;
     metadata?: InputMaybe<Array<InputMaybe<ValueMetadataInput>>>;
-    value?: InputMaybe<Scalars['String']>;
+    payload?: InputMaybe<Scalars['String']>;
```
Comment (renaudAmsellem) [inline apps/data-studio/src/_gqlTypes/index.ts:815]: **question:** Ça n'avait pas été regénéré depuis quelques temps non ?

File: libs/ui/src/components/RecordEdition/EditRecordContent/uiElements/StandardField/StandardFieldValue/ValuesList/MonoValueSelect.test.tsx [hunk 2]
```diff
@@ -348,6 +371,40 @@ describe('<MonoValueSelect />', () => {
             const newColorOption = screen.getByText(`record_edition.select_option${newColor}`);
             await userEvent.click(newColorOption);
 
+            expect(handleSubmitMock).toHaveBeenCalledWith(newColor, attribute.id);
+            expect(screen.getByTestId(attribute.id).innerHTML).toMatch(new RegExp(newColor));
+        });
+
+        it.only('should display the option and create it with list update', async () => {
```
Comment (renaudAmsellem) [inline libs/ui/src/components/RecordEdition/EditRecordContent/uiElements/StandardField/StandardFieldValue/ValuesList/MonoValueSelect.test.tsx:378]: **issue:** oops

File: libs/ui/src/locales/en/shared.json [hunk 3]
```diff
@@ -462,7 +462,8 @@
         "search_not_found": "No match",
         "press_enter_to": "Press enter to ",
         "select_this_value": "Select this value",
-        "select_option": "Select option "
+        "select_option": "Select option ",
+        "create_option": "Create and select option "
```
Comment (renaudAmsellem) [inline libs/ui/src/locales/en/shared.json:466]: **suggestion:** 
```suggestion
        "create_and_select_option": "Create and select option "
```

**Review by philippechevieux — APPROVED** [review]

File: apps/data-studio/src/_gqlTypes/index.ts (hunk 1 above)
Comment (renaudAmsellem) [reply apps/data-studio/src/_gqlTypes/index.ts:815]: Ouais on s'est dit pareil avec Philippe !

**Review by renaudAmsellem — APPROVED** [review]

**Pull request merged**